core/src/main/resources/jenkins/install/pluginSetupWizard_it.properties

```diff
@@ -1,51 +1,50 @@
+installWizard_error_restartNotSupported=Il riavvio non \u00E8 supportato, riavviare manualmente questa istanza
 installWizard_installCustom_title=Per iniziare
 installWizard_installCustom_selectAll=Tutti
 installWizard_installCustom_selectNone=Nessuno
 installWizard_installCustom_selectRecommended=Suggeriti
 installWizard_installCustom_selected=Selezionati
 installWizard_installCustom_dependenciesPrefix=Dipendenze
-installWizard_installCustom_pluginListDesc=Si noti che qui non viene visualizzato l''elenco completo dei plugin. � possibile installare plugin aggiuntivi nel <strong>Gestore plugin</strong> una volta completata l''installazione iniziale. <a href="https://jenkins.io/redirect/installation-wizard" target="_blank">Si veda il Wiki per ulteriori informazioni</a>.
+installWizard_installCustom_pluginListDesc=Si noti che qui non viene visualizzata la lista completo di plugins. \u00C8 possibile installare plugin aggiuntivi nel <strong>Gestore plugin</strong> una volta completata la prima fase di installazione iniziale. <a href="https://jenkins.io/redirect/installation-wizard" target="_blank">Si veda il Wiki per ulteriori informazioni</a>.
 installWizard_goBack=Indietro
 installWizard_goInstall=Installa
 installWizard_installing_title=Per iniziare
 installWizard_installing_detailsLink=Dettagli...
 installWizard_installComplete_title=Per iniziare
-installWizard_installComplete_banner=Jenkins � pronto!
-installWizard_installComplete_bannerRestart=Jenkins � quasi pronto!
-installWizard_pluginsInstalled_message=L''installazione dei plugin � stata completata.
-installWizard_installComplete_message=L''installazione di Jenkins � stata completata.
+installWizard_installComplete_banner=Jenkins \u00E8 pronto!
+installWizard_installComplete_bannerRestart=Jenkins \u00E8 quasi pronto!
+installWizard_pluginsInstalled_message=Installazione dei plugins completata.
+installWizard_installComplete_message=Installazione di Jenkins completata.
 installWizard_installComplete_finishButtonLabel=Inizia a utilizzare Jenkins
-installWizard_installComplete_installComplete_restartRequiredMessage=L''installazione di Jenkins � stata completata, ma alcuni plugin richiedono il riavvio di Jenkins.
-installWizard_installComplete_installComplete_restartRequiredNotSupportedMessage=L''installazione di Jenkins � stata completata, ma alcuni plugin richiedono il riavvio di Jenkins e sembra che quest''istanza non supporti il riavvio automatico. Riavviare manualmente quest''istanza ora per completare l''installazione.
+installWizard_installComplete_installComplete_restartRequiredMessage=Installazione di Jenkins completata, ma alcuni plugin richiedono il riavvio di Jenkins.
+installWizard_installComplete_installComplete_restartRequiredNotSupportedMessage=Installazione di Jenkins completata, ma alcuni plugin richiedono il riavvio di Jenkins e sembra che quest''istanza non supporti il riavvio automatico. Riavviare manualmente quest''istanza ora per completare l''installazione.
```

Comment:
quest''istanza --> questa istanza (you can do that here). Note you have it twice